src/GraphQL.Tests/DI.xUnit/ServiceRegisterExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using GraphQL.DI;
+
+namespace GraphQL.Tests
+{
+    internal static class ServiceRegisterExtensions
+    {
+        //TODO: rename to transient
+        public static void Register<TService>(this IServiceRegister register)
```

Comment:
Again - I kept all names as before changes to minimize diff - the goal of whole this class. @Shane32 I would like to rename `Register` to `Transient` after your review. Another option - remove extension methods at all and use "plain" `Register` methods but I would not.

Reply:
Changing Register to Transient makes sense here.

Reply:
done

Reply:
done